src/chainparams.cpp

```diff
@@ -292,7 +292,7 @@ class CTestNetParams : public CChainParams {
         pchMessageStart[1] = 0x11;
         pchMessageStart[2] = 0x09;
         pchMessageStart[3] = 0x07;
-        nDefaultPort = 18333;
+        nDefaultPort = DEFAULT_TESTNET_PORT;
```

Comment:
For consistency it would be nice to set a constant / enum for regtestnet's port too (would require additional entry in chainparams.h)